pkg/storaged/stratis-details.jsx

```diff
@@ -171,9 +180,27 @@ export function validate_pool_name(client, pool, name) {
 
 export const StratisPoolDetails = ({ client, pool }) => {
     const filesystems = client.stratis_pool_filesystems[pool.path];
+    const blockdevs = client.stratis_pool_blockdevs[pool.path] || [];
 
     const forced_options = ["x-systemd.requires=stratis-fstab-setup@" + pool.Uuid + ".service"];
 
+    function grow_blockdevs() {
+        return for_each_async(blockdevs, bd => pool.GrowPhysicalDevice(bd.Uuid));
+    }
+
+    let alert = null;
+    if (blockdevs.some(bd => bd.NewPhysicalSize[0] && Number(bd.NewPhysicalSize[1]) > Number(bd.TotalPhysicalSize))) {
+        alert = (<Alert key="unused-space"
+                        isInline
+                        variant="warning"
+                        title={_("This pool does not use all the space on its block devices.")}>
+            {_("Some block devices of this pool have grown in size after the pool was created. The pool can be safely grown to use the newly available space.")}
+            <div className="storage_alert_action_buttons">
+                <StorageButton onClick={grow_blockdevs}>{_("Grow the pool to take all space")}</StorageButton>
+            </div>
+        </Alert>);
```

Comment:
These 10 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19079-20230707-110927-1232dadb-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)